src/babel/babel.ts

```diff
@@ -54,13 +54,22 @@ const defaultConfiguration: ImportedConfiguration = (
   existsSync(configurationFile) ? require(configurationFile) : {}
 ) as ImportedConfiguration;
 
+const resolveModuleNames = (config: ImportedConfiguration): Required<Required<ImportedConfiguration>['modules']> => ({
+  boot: config.modules?.boot ?? 'react-imported-component/boot',
+  macro: config.modules?.macro ?? 'react-imported-component/macro',
+  wrapper: config.modules?.wrapper ?? 'react-imported-component/wrapper',
+  native: config.modules?.wrapper ?? 'react-imported-component',
```

Comment:
wondering if so "fine-tune" is required.
I was thinking about letting configuring only the package name, that would solve the case for monorepos, as long as there is nothing to solve for "non-monorepo" ones.

Reply:
just didn't feel right enforcing sub-module named that way. But perhaps, just enforce module topology. aka one must expose a `/boot` sub-module for example, and all one can control is the "name".

Or what about more something like.

```
const bootModule = require.resolve('react-imported-component/boot');

t.stringLiteral(bootModule);
```

that way its also "importing this one" without using module resolutions at build-time.

Reply:
One can implement this functionality at the `setConfiguration` level, resolving up to absolute paths if necessary.